src/scss/_color-use-cases.scss

```diff
@@ -1 +1,8 @@
 @include oColorsSetUseCase(o-card, background, 'pink-tint1');
+@include oColorsSetUseCase(o-card-shadow, shadow, 'pink-tint3');
+@include oColorsSetUseCase(o-card-standout, background, 'pink-tint2');
+@include oColorsSetUseCase(o-card-tag, text, 'claret');
+@include oColorsSetUseCase(o-card-tag-hover, text, 'grey-tint4');
+@include oColorsSetUseCase(o-card-timestamp, text, 'warm-6');
+@include oColorsSetUseCase(o-card-standfirst, text, 'grey-tint5');
```

Comment:
Some of these colours are duplicated as warm or cold whatever names. I'd prefer to use the latter as those are the next palette colour names